src/kOS.Safe/Encapsulation/UserDelegate.cs

```diff
@@ -1,17 +1,18 @@
 ﻿using System.Collections.Generic;
+using kOS.Safe.Execution;
+using kOS.Safe.Compilation;
 
-namespace kOS.Safe.Execution
+namespace kOS.Safe.Encapsulation
```

Comment:
Rationale behind why this moved to a new location:  Previously  UserDelegate had no user-exposed parts inside.  Now it's a kind of KOSDelegate and therefore has suffixes users can access.